static/js/browser2/weather_chart_section.tsx

```diff
@@ -105,11 +114,15 @@ export class WeatherChartSection extends React.Component<
             return;
           }
           const values = {};
+          let provId = "";
           weatherData.forEach((data) => {
             values[data.observationDate] = data.meanValue;
+            if (_.isEmpty(provId)) {
+              provId = data.provId;
+            }
           });
           const sourceSeries = {
-            provenanceDomain: "",
+            provenanceDomain: provId,
```

Comment:
why is domain assigned with id here? Need some name tweek?

Reply:
I updated it to be assigned with actual domain